File: src/DataStructure/LinkedList/Medium/MyLinkedList/javascript/MyLinkedList.test.js
import solution from './MyLinkedList.es3.cjs';
import runTestCase, { cases } from './testCases.js';

test.each(cases.map(c => [c.name, c]))(
  '[Data Structure/Linked List/Medium/[707] Design Linked List] - Case %#',
  (name, testCase) => expect(runTestCase(solution, testCase)).toEqual(testCase.expected),
);

File: src/DataStructure/LinkedList/Medium/MyLinkedList/Singly/python/MyLinkedList_test.py
import pytest
from MyLinkedList.Singly.python.MyLinkedList import MyLinkedList
from MyLinkedList.Singly.python.testCases import runTestCase, cases


@pytest.mark.parametrize('case', cases)
def test(case):
    assert runTestCase(MyLinkedList, case) == case['expected']
